### PR TITLE
capture additional facts

### PR DESCRIPTION
**Pull request by beav**
This commit captures the following new facts:

* list of installed packages
* network interfaces
* running processes
* CPU info, including socket count and flags

### CODE REVIEW

File: pup/utils/fact_extract.py [hunk 1]
```diff
@@ -6,23 +6,25 @@
...
 from insights.parsers.meminfo import MemInfo
+from insights.parsers.ps import PsAuxcww
 from insights.parsers.redhat_release import RedhatRelease
-from insights.parsers.uname import Uname
 from insights.parsers.systemd.unitfiles import UnitFiles
+from insights.parsers.uname import Uname
 from insights.parsers.virt_what import VirtWhat
 from insights.specs import Specs
-from insights.util.canonical_facts import get_canonical_facts, IPs
+from insights.util.canonical_facts import get_canonical_facts
```
Comment (beav) [inline pup/utils/fact_extract.py]: apologies for sorting these in this PR, it makes the changes harder to read :eyeglasses: 

`PsAuxcww` and `IPAddr` were added, and `IPs` was removed

**Comment by SteveHNH** [comment]
This one looks good. Just holding off until inventory is ready. We'll merge it when that happens :+1:

File: pup/utils/fact_extract.py (hunk 1 above)
Comment (SteveHNH) [reply pup/utils/fact_extract.py]: no worries!

**Comment by beav** [comment]
> This one looks good. Just holding off until inventory is ready. We'll merge it when that happens +1

thx for review!

This PR uses the old style of fact recording still and should be compatible with today's inv service. I was gonna make a new PR to switch all of the facts over to the new style after things are working with the old style (only like 2 or 3 pieces of info left at this point).

**Comment by SteveHNH** [comment]
@beav I just talked to Derek with inventory. He already has the new style in CI :P. Might need to go ahead and submit the new PR

**Comment by beav** [comment]
> @beav I just talked to Derek with inventory. He already has the new style in CI :P. Might need to go ahead and submit the new PR

np, will do

**Comment by beav** [comment]
This has been redone with inv service changes in https://github.com/RedHatInsights/insights-pup/pull/62

**Pull request closed without merging**